### DISCUSSION TITLE
Doubts abot fail2ban

### DISCUSSION THREAD

**Discussion by gbertolottiKS** [nextcloud/all-in-one#3667] [Questions] [answered]
Hi.
We want to turn on fail2ban, but I'm not sure what is the best road to follow.
We see the documentation (https://github.com/nextcloud/all-in-one#fail2ban) on the README, but still we have some doubts:

1. What is the preferred way to install it? Downloading and installing it manually or using the community container? Are they equivalent?
2. Does it act only at the level of Nextcloud or the whole operating system? 
3. The community container needs only to be added to AIO, and it's done? 

Thanks in advance

**Comment by szaimen** [comment, marked as answer]
> * What is the preferred way to install it? Downloading and installing it manually or using the community container? Are they equivalent?

Yes, they should be equovalent

> * Does it act only at the level of Nextcloud or the whole operating system?

Whole OS afaik

> * The community container needs only to be added to AIO, and it's done?

Yes this is the idea
